client/src/components/pages/common/Login/index.js

```diff
@@ -1,5 +1,125 @@
 import React from 'react';
+import './index.css';
```

Comment:
just for organizing add this line at the end separated by blank line after component imports